kanidm_client/src/lib.rs

```diff
@@ -65,6 +65,8 @@ struct KanidmClientConfig {
     verify_ca: Option<bool>,
     verify_hostnames: Option<bool>,
     ca_path: Option<String>,
+    // prompt users for tokens instead of just bailing
+    prompt_user_token: Option<bool>,
```

Comment:
Is there really a reason to make this configurable? To login you have to do an "interactive" process anyway from the cli tool. It also seems like a "layer" violation (config in the client, but it's used by the cli tool).

I think I'd rather remove the prompt_user_token bool, and just have the menu selector instead.

Reply:
I figured some beardy types wouldn't want to be prompted like that, so made it an optional thing. Is there a cli tool config file? I thought the client config was the place for it 😄 

Happy to take that config part out if you'd prefer 😄

Reply:
I would rather remove this because it should either be a cli switch during login to remove the interactivity, but also, I don't want this to "become a scripting" tool. That's what the API should be for, and having people add tools like this to scripts just causes people to complain if we make future usability changes/improvements. I don't really want to lock us into that, so for now, I think remove the option because it sohuld be "interactive only". :)

Reply:
Sold, will remove it this evening :)